x-pack/plugins/task_manager/server/routes/health.ts

```diff
@@ -139,6 +139,12 @@ export function healthRoute(params: HealthRouteParams): {
   router.get(
     {
       path: '/api/task_manager/_health',
+      security: {
+        authz: {
+          enabled: false,
+          reason: 'This route is opted out from authorization',
```

Comment:
We want this endpoint to require authorization but it would be a breaking change.. we have this GitHub issue to track the request: https://github.com/elastic/kibana/issues/136157